src/commands.jl

```diff
@@ -45,7 +45,14 @@ function propagate_exception!(state::DebuggerState, exc)
     rethrow(exc)
 end
 
+function assert_is_toplevel_frame(state)
+    state.level == 1 && return true
```

Comment:
I suppose the alternative is to set `state.level = 1`. But perhaps the error is better.